### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by JosiasVasquez** [agata/dl.biscuit#46]
This app is the most accessible multi account app I found, I was test a lot of apps, but all apps were not accessible with screen readers. Biscuit browser was the one to help me to have several accounts of several services in one place.

But some features can be improve.

1. Tag the buttons of the toolbar, my screen reader only said me button all the time. I push this buttons but I don't know what this buttons do.

**Comment by JosiasVasquez** [comment]
I am web accessibility specialist, if you want I can help to audit an remediate the app.
